package.json

```diff
@@ -168,10 +179,18 @@
         "command": "mdb.createPlayground",
         "title": "MongoDB: Create MongoDB Playground"
       },
+      {
+        "command": "mdb.refreshPlaygrounds",
```

Comment:
Ah I think this one makes it into the command palette.
```
...
"editor/title": [{
  "command": "mdb. refreshPlaygrounds",
  "when": "false"
}
...]
```
Is it something we want in the command palette? In that case maybe we could add a command for that which has the `MongoDB:` prefix

Reply:
Added additional command for refreshing playgrounds from the command palette in addition to one is being called from the tree view. Plus test for it.